src/lib.rs

```diff
@@ -360,10 +360,62 @@ where
     ///
     /// # Example
     ///
-    /// ```ignore
-    /// table.set_on_sort(|siv: &mut Cursive, column: BasicColumn, order: Ordering| {
+    /// ```rust
```

Comment:
Any reason why these are modified?

Reply:
I saw that one of the tests ignored from the beginning was solved and I modified it to solve this one as well